.chloggen/mongodb_add_address_res_attr.yaml

```diff
@@ -0,0 +1,30 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
Out of ignorance: It's considered a breaking change when metrics are switched to be enabled by default. Since we're adding a resource attribute that is now enabled by default, should this also be a breaking change? Or is it so little extra data that it doesn't merit the breaking label?

I guess counterpoint, adding attributes to metrics isn't considered a breaking change, so I'm not sure where the line in the sand is. (I'm also not finding documentation on this, either way).

Reply:
We don't have strict guidelines for the resource attributes. I'd say adding a new resource attribute is not a breaking change because it doesn't change cardinality. Removing or changing one should be considered a breaking change, though.

Happy to hear other opinions. cc @djaglowski

Reply:
I've generally considered it a breaking change because it breaks time series that are based on all attributes.

Reply:
By breaking time series you mean just the interrupted continuity, no other user-facing effect, right?

Reply:
I would merge this as is, assuming that's the case ^.  Let me know if there are any other consequences for the end user that I miss, I'll change it.